github/client.go

```diff
@@ -134,6 +134,23 @@ func (client *Client) CreatePullRequest(project *Project, params map[string]inte
 	return
 }
 
+func (client *Client) GetPullRequestUrlsForHead(headProject *Project, branch string) (pr []PullRequest, err error) {
+	api, err := client.simpleApi()
+	if err != nil {
+		return
+	}
+
+	res, err := api.Get(fmt.Sprintf("repos/%s/%s/pulls?head=%s:%s", headProject.Owner, headProject.Name, headProject.Owner, branch))
```

Comment:
The `owner` and `repo` values in `repos/:owner/:repo/...` URLs should not be the head project. Instead, it should be the "base" project (which is considered to be the "canonical" version of the project at which issues and PRs are opened).